data/moves.js

```diff
@@ -14550,6 +14550,9 @@ exports.BattleMovedex = {
 			if (myItem) {
 				target.setItem(myItem);
 				this.add('-item', target, myItem, '[from] move: Trick');
+			}else {
+				target.setItem(myItem);
+				this.add('-enditem', target, 'Air Balloon');
```

Comment:
An analogal message should probably be sent if the target pokémon doesn't have an item, as well.

Also, this is missing the source effect parameter, which the client currently doesn't handle properly (it assumes the balloon popped).